bazel/thirdparty/krb5.BUILD

```diff
@@ -0,0 +1,28 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"]),
+)
+
+configure_make(
+    name = "krb5",
+    autoreconf = True,
+    autoreconf_options = ["-ivf ./src"],
+    configure_command = "./src/configure",
+    configure_in_place = True,
+    configure_options = [
+        "--srcdir=./src",
+        "--disable-thread-support",
+        "--without-netlib",
+        # TODO when building the static library the linker is exiting with a
```

Comment:
Can we put `TODO(bazel)` in all the TODOs in this PR so they are more greppable?

Reply:
Fixed